lib/integration/esbuild.js

```diff
@@ -39,7 +39,12 @@ export function esbuild(options) {
    * @param {OnLoadArgs} data
    */
   async function onload(data) {
-    var doc = String(await fs.readFile(data.path))
+    /** @type {string} */
+    var doc =
+      data.pluginData && data.pluginData.contents
```

Comment:
`pluginData` is causing some headaches with types as `type-coverage` isn't happy with it being `any`.

I've tried changing the type of `data` to `{OnLoadArgs & {pluginData?: {content?: string}}}` to no avail. Looks like typedoc isn't liking joining two objects?

Reply:
Yep that works! Pushed the commit.